deploy/templates/nstemplatetiers/advanced/ns_code.yaml

```diff
@@ -50,12 +50,12 @@ objects:
   metadata:
     labels:
       toolchain.dev.openshift.com/provider: codeready-toolchain
-    name: user-toolchain-che-edit
+    name: user-rbac-edit
```

Comment:
in https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/nstemplatetiers/advanced/ns_dev.yaml#L18-L23 the name is `user-edit`.

Reply:
It's different `RoleBinding`. In that `RoleBinding` we set the `ClusterRole` `edit` to the user, here we set the `rbac-edit` `Role` to the user

Reply:
ok, my bad. 
I understand your intent and I agree with it, but how can we easily differentiate between `user-edit` and `user-rbac-edit` ? IMO `rbac` in the `user-rbac-edit` role name does not help/clarify much :(

Reply:
my understanding of the format is:
`<who-gets-the-permission>-<name-of-role>`

in the same way, we will able to set the `RoleBindings` for collaborators. So in a case I wanted to invite you to my team namespace (as an "administrator" - giving you the "full access"), I would create `RoleBindings`:
`xcoulon-at-redhat-com-edit` and `xcoulon-at-redhat-com-rbac-edit`
This is the template I would use for it: https://gist.github.com/MatousJobanek/81f552e889f5c8b6756c251b2a98074f 
If I wanted to give you only view permissions, then I would create `xcoulon-at-redhat-com-view` RB

so we could change the names to: `${USERNAME}-edit` `${USERNAME}-rbac-edit`, but I wanted to have a way to differentiate between collaborators and the owner of the namespace. So collaborators'  RBs would have names:
`<name-of-collaborator>-<name-of-role>`
but the owner of the namespace has always:
`user-<name-of-role>`

> IMO rbac in the user-rbac-edit role name does not help/clarify much :(

`user-rbac-edit` is a `RoleBinding` not a `Role`
The `Role` has the name `rbac-edit` expressing that the role is giving you edit permissions (all CRUD verbs) for the RBACs resources.

But if you have some better names, I'm open for a discussion :-)

Reply:
ok, fine.

Reply:
thanks for the clarification.